src/node_errors.cc

```diff
@@ -375,8 +375,13 @@ static void ReportFatalException(Environment* env,
     }
 
     if (!env->options()->trace_uncaught) {
-      FPrintF(stderr, "(Use `node --trace-uncaught ...` to show "
-                      "where the exception was thrown)\n");
+      std::string argv0;
+      if (!env->argv().empty()) argv0 = env->argv()[0];
+      if (argv0.empty()) argv0 = "node";
+      FPrintF(stderr,
+              "(Use `%s --trace-uncaught ...` to show where the exception "
+              "was thrown)\n",
+              fs::Basename(argv0, ".exe"));
```

Comment:
Our build files say we compile with `-std=gnu++1y` so you could use `std::filesystem::path(argv0).stem()` for this, although library support might be an issue...

`Basename()` doesn't handle forward slashes on Windows.

Reply:
> `Basename()` doesn't handle forward slashes on Windows.

Can you expand on that? `kPathSeparator` is `\\/` on Windows, so it should find the last path separator regardless of whether it’s a forward or backwards slash?

Reply:
I missed that `kPathSeparator` is a string on Windows. It's `char` on other platforms.